application/console/migrations/m181207_144943_create_invite_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use yii\db\Migration;
+
+/**
+ * Handles the creation of table `invite`.
+ */
+class m181207_144943_create_invite_table extends Migration
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function safeUp()
+    {
+        $this->createTable(
+            'invite',
+            [
+                'id' => 'pk',
+                'user_id' => 'int(11) not null',
```

Comment:
Do we want/need to allow users to have multiple invites?

Reply:
No. Is there an easy way to enforce 1:1?

Reply:
After further thought, I realize I could make `user_id` unique. However, to keep moving, I will not change anything here. The current business logic should prevent multiple invites for any user. Actually, to be precise, it's not possible for multiple _non-expired_ invites per user. Presently, there is no garbage collection. I'll make a task card to write a cron script to clean up expired records.